content/blog/branch-patterns-automate-prs.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Branch patterns and automating PRs
+date: '2022-04-14T07:00:00.000Z'
+author: James Perkins
+prev: content/blog/read-only-tokens-content-anytime.md
+---
+
+When working on a git backed site you don’t normally want to work directly on your main branch. Ideally, you want to be able to finish your post or site update, check if it looks good, and schedule the release. A good branch pattern and scheduler is key to making this work. In this post, we are going to cover two different options to automate your PRs and an example of the branch pattern I use with Tina.
+
+## Branch Pattern
+
+Branching patterns are very much a personal or business decision, however it usually comes down to how we work on a feature or fix, where we stage the code, and finally when we deploy it. 
+
+My preferred branch pattern looks like the following:
+
+### Permanent Branches:
+
+**main** - This holds the production site that is live and is also the source of truth.
+
+**staging** - This holds the current planned work that is ready for release but may also be a copy of the main branch.
+
+### Temporary Branches:
+
+**feature-name** - Used for a new feature on a site or application.
+
+**bug-fix-name** - Used to fix a bug that was found.
+
+**blog-post-name** - Used for a blog post article. 
+
+When work on a temporary branch is complete, the pull request will go to staging. Here, it will be tested and determined if everything works as expected. A pull request will then be opened to production. Once the production release is complete, the temporary branch will be removed.
+
+## PR Scheduler
+
+PR Scheduler is a GitHub integration that can be installed directly within your GitHub repositories. It was built by [Tom Kadwill](https://tomkadwill.com/) with the goal to make it easy to schedule pull requests. Instead of having to write your own GitHub action, you can write a comment in your pull request and the application will take care of it for you.
+
+### How to install
+
+1. Open [PR Scheduler's GitHub App page](https://github.com/apps/pr-scheduler).
+2. Click the 'Install' button
+3. Select whether to install PR Scheduler on all repositories or only specific repositories. Then click 'Install'.
+
+### How to schedule a pull request
+
+Now, the PR Scheduler can now be used to schedule any of your pull requests. To initiate,do the following:
+
+1. Open  the pull request that you want to schedule.
+2. Add a new comment with DD/MM/YYYTHH:MM for example `@prscheduler 05/04/2022T14:00`
+3. PR Scheduler will respond back telling you it's ready.
+
+
+That's it! Now when that time comes, your PR will be merged. If you make a mistake with the time or date, just run the same command and it will reschedule.
+
+![Example Image of PR Scheduled](https://res.cloudinary.com/forestry-demo/image/upload/v1649865121/blog-media/branch-automate-pr/Screen_Shot_2022-04-12_at_7.34.54_AM.png)
+
+## Github Actions
```

Comment:
I think it might be clearer to specifically call out Option 2: Github Actions in the section title or in the opening paragraph of this section, just to make it clear that we've finished option 1 and now we are showing a second option